components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -201,6 +208,32 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         response.sendRedirect(getRedirectURL(redirectURL, request));
     }
 
+    /**
+     * If postLogoutRedirectUri is send in Logout request parameter then set it as redirect URL
+     * @param redirectURL
+     * @param opBrowserStateCookie
+     * @return
+     * @throws UnsupportedEncodingException
+     */
+    private String redirectToPostLogoutRedirectUri(String redirectURL, Cookie opBrowserStateCookie)
```

Comment:
Better to change it to "generatePostLogoutRedirecctUrl

Reply:
okay